pygmt/datasets/samples.py

```diff
@@ -360,6 +362,21 @@ def load_mars_shape(**kwargs):
     return data
 
 
+def _load_sample_compositions():
+    """
+    Loads the remote file @ternary.txt.
+
+    Returns
+    -------
+    data : pandas.DataFrame
+        The data table with columns "a", "b", "c", "z".
```

Comment:
Is there any information on what the compositions are? I don't see it in the actual `ternary.txt`, but going from your galley example, I'm assuming it's "water", "air" and "limestone" (not sure what the "z" stands for). Would it be better for the column data to be labeled as such?

Reply:
The first 3 columns are clear, however, since z is not used in any example I'm not sure what stands for.

Reply:
We can skip the "z" column if it's not used.

Reply:
You mean to leave z as is @seisman?

Reply:
I mean using `names=["water", "air", "limestone"], usecols=(0, 1, 2)` to only load the first three columns.

Reply:
Maybe I am missing here some context.

Do I understand it correctly that this dataset should be used for the `ternary` gallery example in PR #2138 which is based on the corresponding GMT upstream example https://docs.generic-mapping-tools.org/dev/ternary.html#examples? In this case, the fourth column of the dataset is need for the color-coding, i.e., the fill color of the symbols, currently circles (**-C** or``cmap=True``).

This dataset seems to be taken from this MATLAB file exchange record https://de.mathworks.com/matlabcentral/fileexchange/7210-ternary-plots?s_tid=mwa_osa_a.
There, the fourth column is listed as the "effective dielectric permittivity" (following the header of `termain.m`).

At the moment I am wondering if the column names are correct: We read 0->water, 1->air, 2->limestone. 
In the MATLAB file, they say 1->solid/limestone 2->water 3->gas/air (following the header of `termain.m`)?